packages/storybook/src/nimble/number-field/number-field-matrix.stories.ts

```diff
@@ -143,15 +194,17 @@ const notRequiredState = requiredVisibleStates[0];
 const interactionStatesHover = cartesianProduct([
     disabledReadOnlyStates,
     appearanceStates,
+    fullBleedStates,
     [notRequiredState],
     [hideStepStateStepVisible],
     [valueStatesHasValue],
     [errorStatesNoError, errorStatesErrorWithMessage]
 ] as const);
 
 const interactionStates = cartesianProduct([
-    disabledReadOnlyStates,
+    onlyDisabledAbsentStates,
```

Comment:
Note: This is fixing an oversight in my previous change (#2600). These interaction states should not be run on disabled controls, so I've updated the states that get passed here.